### PR TITLE
refactor css variables for welcomeOverlay

### PR DESCRIPTION
**Pull request by Kalmaegi**
For https://github.com/microsoft/vscode/issues/165169

- [x] src/vs/workbench/contrib/welcomeOverlay/browser/welcomeOverlay.ts

### CODE REVIEW

**Comment by weartist** [comment]
@chrmarti  Hi, how do I request a review for this PR~

**Review by chrmarti — DISMISSED** [review]

File: src/vs/platform/theme/common/colorRegistry.ts [hunk 1]
```diff
@@ -510,6 +510,12 @@ export const breadcrumbsFocusForeground = registerColor('breadcrumb.focusForegro
 export const breadcrumbsActiveSelectionForeground = registerColor('breadcrumb.activeSelectionForeground', { light: darken(foreground, 0.2), dark: lighten(foreground, 0.1), hcDark: lighten(foreground, 0.1), hcLight: lighten(foreground, 0.1) }, nls.localize('breadcrumbsSelectedForeground', "Color of selected breadcrumb items."));
 export const breadcrumbsPickerBackground = registerColor('breadcrumbPicker.background', { light: editorWidgetBackground, dark: editorWidgetBackground, hcDark: editorWidgetBackground, hcLight: editorWidgetBackground }, nls.localize('breadcrumbsSelectedBackground', "Background color of breadcrumb item picker."));
 
+/**
+ * welcomeOverlay background color.
+ */
+export const welcomeOverlayBackground = registerColor('welcomeOverlay.background', { light: '#FFFFFF85', dark: '#00000085', hcDark: null, hcLight: null }, nls.localize('welcomeOverlayBackground', "welcomeOverlay Background color."));
```
Comment (aeschli) [inline src/vs/platform/theme/common/colorRegistry.ts:516]: The color definition should stay in the component, if possible.

**Review by aeschli — CHANGES_REQUESTED** [review]

File: src/vs/workbench/contrib/welcomeOverlay/browser/welcomeOverlay.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (aeschli) [inline src/vs/workbench/contrib/welcomeOverlay/browser/welcomeOverlay.css:2]: this should go into the existing src/vs/workbench/contrib/welcomeOverlay/browser/media/welcomeOverlay.css

**Review by aeschli — CHANGES_REQUESTED** [review]

**Review by aeschli — APPROVED** [review]

**Review by chrmarti — APPROVED** [review]
Reapproving, thanks @weartist & @aeschli !

**Pull request merged**